purge-build-queue-by-name.groovy

```diff
@@ -1,5 +1,11 @@
 //Purge build queue by name
 
+// find tasks by name which contains string pattern REPLACEME
 import hudson.model.*
 def q = Jenkins.instance.queue
-q.items.findAll { it.task.name.startsWith('REPLACEME') }.each { q.cancel(it.task) }
+q.items.findAll { it.task.name.contains('REPLACEME') }.each { println it.task.name }
```

Comment:
```suggestion
Queue.getInstance().items.findAll { it.task.name.contains('REPLACEME') }.each { println it.task.name }
```

Since this does not have to reuse the `q` to cancel task, it can be reduced to one line.